sig-TEMPLATE/sig-charter-template-prefilled.md

```diff
@@ -0,0 +1,33 @@
+# SIG YOURSIG Charter
```

Comment:
This is probably fine for now, since we don't have a golden example, but it would be better if one of our existing SIGs had a beautiful example of a charter that we could point to instead. Folks can just as easily copy-paste and update one of them.

Reply:
True!

Let me help with a specific example. But let's not block on it pls